src/fields/abstractField.js

```diff
@@ -72,6 +84,34 @@ export default {
 	},
 
 	methods: {
+		deleteDataOnHide() {
+			let field = this.schema;
+			let visible = field.visible;
+
+			if (isFunction(field.visible)) {
+				visible = field.visible.call(this, this.model, field, this);
+			}
+
+			if (isNil(field.visible)) {
+				visible = true;
+			}
+			// if the schema formOptions includes a deleteDataOnHide attribute:
+			// 1. we check if visibility became false on the field. If so, delete the model from that field
+			// 2. if became visible and we have an initial, we can set that initial back if the field not in model
+			if (this.formOptions.deleteDataOnHide) {
```

Comment:
Let's allow for individual values to keep their value when visible with a schema option. There might be certain cases where globally we want to delete fields on invisible (so we set `deleteDataOnHide`) but certain fields we don't want to delete and can set it on that field (maybe `schema.keepOnHide`).